packages/libs/react-ui/src/components/NavHeader/NavHeader.css.ts

```diff
@@ -210,13 +207,15 @@ export const selectClass = style([
 
 export const chevronIconClass = style([
   sprinkles({
+    height: '$9',
```

Comment:
would it make more sense to position it to the top, bottom, and right? Right now if the height doesn't match the select component, it will still not be centered